src/torchrunx/agent.py

```diff
@@ -116,9 +113,16 @@ def main(launcher_agent_group: LauncherAgentGroup):
     hostname = launcher_payload.hostnames[agent_rank]
     worker_world_size = launcher_payload.worker_world_size
     worker_global_ranks = launcher_payload.worker_global_ranks[agent_rank]
-    worker_log_files = launcher_payload.worker_log_files[agent_rank]
     num_workers = len(worker_global_ranks)
 
+    logger = logging.getLogger(f"torchrunx.{launcher_payload.hostnames[agent_rank]}")
+    logger.setLevel(logging.DEBUG)
+    socketHandler = logging.handlers.SocketHandler(
+        host=logger_hostname,
+        port=logger_port,
+    )
+    logger.addHandler(socketHandler)
```

Comment:
Wondering why we're doubling this (see also `entrypoint`). So that we can propagate the `torchrunx.agent.main` logs?